src/main/java/use_cases/ExportOutputBoundary.java

```diff
@@ -0,0 +1,8 @@
+package use_cases;
+
+import java.util.List;
+
+public interface ExportOutputBoundary {
+
+    public String prepareSuccess();
```

Comment:
I think it makes sense we only have prepare success message here! We may not have any situation that does not export successfully.